generate-dockerfile.py

```diff
@@ -14,206 +20,374 @@
     ("io.openshift.tags", "security openscap scan"),
     ("install", "docker run --rm --privileged -v /:/host/ IMAGE sh /root/install.sh IMAGE"),
     ("run", "docker run -it --rm -v /:/host/ IMAGE sh /root/run.sh"),
-    ("help", "docker run --rm --privileged -v /usr/bin:/usr/bin -v /var/run:/var/run -v /lib:/lib -v /lib64:/lib64 -v /etc/sysconfig:/etc/sysconfig IMAGE sh /root/help.sh IMAGE")
-]
-packages = [
-    "bzip2",
-    "wget"
+    ("help", "docker run --rm --privileged -v /usr/bin:/usr/bin -v /var/run:/var/run -v /lib:/lib -v /lib64:/lib64 -v /etc/sysconfig:/etc/sysconfig IMAGE sh /root/help.sh IMAGE"),
 ]
+packages_for_cve_feed = {
```

Comment:
I tested both base images, Fedora and RHEL7, there is no `bzip2` package installed by default. I think there should be no problem and we can add `bzip2` package into the `packages_for_cve_feed`.

Reply:
When I try this on my machine, the baseimage already contains bzip2 at install-time, and when removing it, it wants to also remove `Python` and `openscap-daemon` (it is the same behaviour as on my Fedora 27, see below). So the conclusion is to keep `wget` and `bzip2` installed as they occupy little space and may be useful in the future.
```
$ sudo dnf remove bzip2                                                                                                                                                       
Dependencies resolved.
=======================
Package                                
=======================
Removing:
bzip2            ...
Removing dependent packages:
 flatpak-builder                       
...
 zstd                ...
Transaction Summary
======================
Remove  144 Packages
...
```